apps/fluent-tester/src/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -33,5 +33,7 @@ export const AvatarTest: React.FunctionComponent = () => {
   const description =
     'AvatarView is a visual representation of a user, entity, or group. If an image is supplied, it is cropped to a circle of the requested size. If an image is not supplied, initials are extracted from the given name and email address provided and displayed on a colorful background.';
 
-  return <Test name="Avatar Test" description={description} sections={avatarSections} status={status} />;
+  const spec = 'https://github.com/microsoft/fluentui-react-native/blob/main/packages/components/Avatar/SPEC.md';
```

Comment:
Should this URL be stored somewhere else, in some sort of resource package or something? Or at the very least maybe this should be a sort of URL builder function somewhere.

I'm thinking about how these links will be a pain to maintain when/if any of this code ends up moving somewhere else, even within the same project.

Reply:
Possibly a good follow-up, but probably hard to do on Joanna's last day?

Reply:
Our link follow test /might/ be savvy enough to pick up on it. But yeah it'd be more robust to have a builder for sure